gallery/how_to/work_with_microtvm/install_zephyr.rst

```diff
@@ -0,0 +1,34 @@
+Install the Prerequisites
+----------------------------
+
+    .. code-block:: bash
```

Comment:
The same question here. Probably we should design this file as a documentation and add more text information? Because it looks more like a bash script for now.

Reply:
There is now a comment clarifying this.